pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -93,6 +93,9 @@ type APIClient struct {
 	// Cl holds the main kubernetes client
 	Cl kubernetes.Interface
 
+	// CRDClient holds the extension kubernetes client
+	CRDClient clientset.Interface
```

Comment:
I named it `CRDClient` because we can only access CRD metrics from this client but in theory, it's a client for api-extensions.